### PR TITLE
Re-package with fully qualified package names

### PR DESCRIPTION
**Pull request by 0ortmann**
###  :notebook_with_decorative_cover: Description

- Update all package versions to 2020.11.26 to prepare all packages for release
- Restructure package folder structure
- Update Makefiles and CI jobs

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/threatbus](https://docs.tenzir.com/threatbus), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

File-by-file.

Try the following on your machine (all in a virtual env):

- `make clean` -> remove previous artifacts, if any
- `make dev-mode`
- configure some plugins and start Threat Bus 
- -> expectation: all can be started
- `make unit-tests`
- -> expectation: all pass

If that worked, we gonna do the exact same thing for different install targets. *Remove your virtual env, setup a new one and source it*, then:
- `make clean`
- `make install`
- repeat steps from above (run it with config, run unit tests)

Again, if that worked, *clean your virtual env, setup a new one and source it*. Then
- `make clean`
- `make dist` -> that produces a bunch of `dist/` folders, both in top-level `threatbus` as well as in each plugin directory
- install Threat Bus and plugins from the created `dist/XXX.tar.gz` artifacts via `pip install /path/to/dist/XXX.tar.gz`
- repeat steps from above (run it with config, run unit tests)

### CODE REVIEW

**Comment by 0ortmann** [comment]
@satta With this PR we can build packages from within their respective source directory instead of repository root.

**Comment by satta** [comment]
Works for me, with the exception of the code dependent on `broker`. Also, I had to install librdkafka and the Python dev headers, but that's hardly Threat Bus's responsibility. :+1:

File: plugins/apps/threatbus_misp/Makefile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.PHONY: test
+test: unit-tests
+
+.PHONY: unit-tests
+unit-tests:
+	python -m unittest discover .
+
+.PHONY: clean
+clean:
+	find . -type d -name __pycache__ -exec $(RM) -r {} \;
```
Comment (mavam) [inline plugins/apps/threatbus_misp/Makefile:20]: Not quite sure where `RM` comes from, but might not be important for this review.

**Review by mavam — APPROVED** [review]
I haven't tested this, but I went through the code. It looks reasonable.

The whole `$(MAKE) -C PLUGIN` doesn't seem to scale, but that's out scope for this review.

File: plugins/apps/threatbus_misp/Makefile (hunk 1 above)
Comment (0ortmann) [reply plugins/apps/threatbus_misp/Makefile:20]: `$(RM)` is a Makefile [variable](https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html).

**Pull request merged**